Simplenote/TagListViewController+Swift.swift

```diff
@@ -282,3 +286,17 @@ extension TagListViewController: SPTextFieldDelegate {
         !menuShowing
     }
 }
+
+
+// MARK: - Appearance
+//
+extension TagListViewController {
+    @objc
+    func refreshTableRowsActiveStatus() {
+        for row in 0..<tableView.numberOfRows {
+            let tableRow = tableView.rowView(atRow: row, makeIfNecessary: false) as? TableRowView
+            tableRow?.isActive = isActive
+        }
+        tableView.reloadSelectedRow()
```

Comment:
Out of curiosity, why do we need to reload the selected row?

Reply:
The cell itself also needs to be updated and not only the row